.github/workflows/nightly-release.yml

```diff
@@ -6,6 +6,13 @@ on:
 
 jobs:
   linux:
+    strategy:
+      matrix:
+        target: 
+          - x86_64-unknown-linux-gnu
```

Comment:
this is for the nightly release. Can you explain the question?

Reply:
My question was about your comment:
```
# TODO: enable after https://github.com/rust-lang/cargo/issues/4133 is resolved
          # - aarch64-unknown-linux-gnu
          # - i686-unknown-linux-gnu
```
If cross-compilation isn't possible in one CI step, could we have several?

Reply:
cross-compilation is not possible even with several steps unfortunately

Reply:
In [go-zerokit-rln](https://github.com/waku-org/go-zerokit-rln/blob/master/scripts/build.sh#L14-L28), to avoid dealing with cargo and linkers, i used `cross-rs` for crosscompiling zerokit with no issues except for some architectures like MIPS .

Reply:
Yes, we'll address that in a follow up pr :)